src/clj_github/repository.clj

```diff
@@ -195,3 +198,21 @@
     (fetch-body! client {:path   url
                          :method :put
                          :body   params})))
+
+(defn clone
+  "Download github repository and put content on destination path"
+  ([client org repo dest]
```

Comment:
I'm wondering whether we should force the client to set a destination directory. The function could just return the path of the temporary directory. Wdyt?

Reply:
Sounds good. Initially, I had kept the destination param to not add a breaking change on common-serverless. But I think it's nice to return the path